src/index.js

```diff
@@ -154,6 +155,110 @@ VirtualMachine.prototype.loadProject = function (json) {
     this.runtime.setEditingTarget(this.editingTarget);
 };
 
+VirtualMachine.prototype.toPrettyJSON = function (testing) {
```

Comment:
This should be handled by the client and not the VM. `toPrettyJSON` sounds like a fine thing for the playground or GUI to do.